### ISSUE TITLE
Extension of #8114, problem with uploading profile data from iAPS Dev

### ISSUE THREAD

**Issue by oddst** [nightscout/cgm-remote-monitor#8119]
**If you need support for Nightscout, PLEASE DO NOT FILE A TICKET HERE**
For support, please post a question to the "CGM in The Cloud" group in Facebook
(https://www.facebook.com/groups/cgminthecloud) or visit the WeAreNotWaiting Discord at https://discord.gg/zg7CvCQ

**Describe the bug**
Uploading basic settings as a profile in Nightscout have failed when Nightscout have been build with 15.0.0 DEV after 12th of October, and for the Master 15.0.0 available now. No error message in iAPS, both using the Development Upload function and individual setting change - like Basal.  
**To Reproduce**
1. Go to iAPS > Settings > Debug = On > Upload
2. Check in Nightscout > Profile Editor

**Expected behavior**
The basic settings like target value/range, Basal, CarbRatio and InsulinSensitivify should have been uploaded and displayed.

**Screenshots**

**Your setup information**
- What version of Nightscout 15.0.0 Master + 15.0.0 DEV from October 12th
- What type of CGM, and how do you get your data there? N/A
- Is your issue specific to a browser (Firefox/Safari/Chrome?) No or a device (Android phone, etc.)? iPhone, but since this is also the situation locally in Nightscout, it is assumed to be caused by Nightscout...!

**Additional context**

**Comment by oddst** [comment]
This bug seem to have been fixed with todays retracts in NS15 DEV...!!!

---

I have upgraded to todays NS15 DEV, since it now is different from NS15 Master (that still is not changed). 

I double-checked my running iAPS version, but it is still the latest iAPS DEV (built yesterday, and there are no commits after that). 

My main looper is in the US and still sleeping, but I tested the following myself:

- I entered the Nightscout Profile Editor, made a change to DIA and Saved it - received a Success reply from Nightscout. Good..!

- I updated all my iAPS settings, to be the same of my running looper in the US, connected iAPS to Nightscout and turned on Update. Changed one Basal settings, save to pump, back to settings and close. The change was updated in Nightscout Profile Editor. Good..!

- I used the Upload button in Settings>Development in iAPS. The settings was uploaded to Nightscout Profile Editor...! Good..!

- Changed the Timezone field directly in Nightscout Profile Editor (due to me being in Norway, and the looper being in the US). Feedback from Nightscout was "Status: success", and the update was there even viewed from a new device. Good..!

I do believe this NS15 DEV change has reverted/solved the problem that arose after a code change in NS15 DEV on October 12th. More testing probably needs to be done by the Nightscout development team, to assure no negative effects of this change...!

Thanks a lot to Milos, Ben and others, that fixed this quickly. Now we only need for NS15 Master to be updated accordingly...!
